src/cosmos/CosmosTools.ts

```diff
@@ -53,11 +56,22 @@ export class CosmosTools implements EdgeCurrencyTools {
     )
     this.methods = methods
     this.registry = registry
+    const { data, name, url } = this.networkInfo.chainData
+    this.chainData = data
+    const chainUpdater = new ChainRegistryFetcher()
+    chainUpdater
+      .fetch(url)
+      .then(() => {
+        this.chainData = chainUpdater.getChain(name)
```

Comment:
Something makes me uneasy about knowing key derivation can be dependent on a network call.  What would we need updated from the network that would change from the hardcoded json?

Reply:
The minimum fees were the main concern. I can see hardcoding the derivation as preferred I'll re-add that to the infos